ucm2/sof-soundwire/rt1318.conf

```diff
@@ -41,5 +41,7 @@ SectionDevice."Speaker" {
 	Value {
 	      PlaybackPriority 100
 	      PlaybackPCM "hw:${CardId},2"
+	      PlaybackMixer "default:${CardId}"
+	      PlaybackMixerElem "rt1318 DAC"
```

Comment:
Use `PlaybackMixerElem "${var:SpeakerMixerElem}"` here.

Reply:
> Use `PlaybackMixerElem "${var:SpeakerMixerElem}"` here.

@perexg Thanks for the review. 
I updated this PR already.